Traceback (most recent call last):
  File "snippet.py", line 15, in <module>
    filename = sys.argv[1]
IndexError: list index out of range
